api/src/main/resources/liquibase.xml

```diff
@@ -180,15 +180,6 @@
         <addForeignKeyConstraint baseTableName="encounter_status" baseColumnNames="patient_id"
                                  constraintName="encounter_status_patient_id_fk"
                                  referencedTableName="patient" referencedColumnNames="patient_id"/>
-        <addForeignKeyConstraint baseTableName="encounter_status" baseColumnNames="encounter_type_id"
-                                 constraintName="encounter_status_encounter_type_id_fk"
-                                 referencedTableName="encounter_type" referencedColumnNames="encounter_type_id"/>
-        <addForeignKeyConstraint baseTableName="encounter_status" baseColumnNames="encounter_id"
```

Comment:
Shouldn't we at least have the FKs on encounter_type and encounter?

Reply:
Yes. My initial thinking was to loosen the restrictions when creating `encounter_status`, without necessarily starting a visit.

Reply:
I think making sure the columns are nullable should be fine. The FKs are semi-important as they create indexes that can be used for joins, i.e., SQL will be somewhat more efficient.